### ISSUE TITLE
remove dependency on tue-filesystem

### ISSUE THREAD

**Issue by PetervDooren** [tue-robotics/ed#115]
We currently have a dependency on the tue-filesystem. In order to make our code usable for others (and keeping our code clean) we should remove this.

Other dependencies we should look at are tue-config and tue-serialisation

**Comment by MatthijsBurgh** [comment]
This should be part of a bigger project. This issue by itself is meaningless

**Comment by PetervDooren** [comment]
I disagree, while work on this issue may require more than 1 PR it is not meaningless.

**Comment by MatthijsBurgh** [comment]
We should have a bigger issue, which focussen on removing all our custom stuff, which could be replaced by more standardized 
ROS stuff. Only removing a dep on `tue-filesystem` is creating work for ourselves if we don't do the rest as well.

Also I don't agree we should focus on removing our custom dependencies. I know we talked about it in the team, I don't think we decided positive on it. So we should talk about it, before creating a list of issues we aren't going to work on. You cleaned the whole shit yourself 🤣
